genapi/genapi.go

```diff
@@ -423,6 +423,13 @@ func (g *GenAPI) serve(h http.Handler, addr string, doTLS bool) {
 	}
 
 	netln := net.Listener(tcpKeepAliveListener{ln.(*net.TCPListener)})
+
+	allowedProxyCIDRsStr, _ := g.ParamStr("--proxy-proto-allowed-cidrs")
```

Comment:
for setting it as an environment variable yeah. I never really figured out how to make multi-params work when set as environment vars =/